pom.xml

```diff
@@ -15,7 +15,7 @@
 	</parent>
 	<groupId>com.newrelic.jenkins</groupId>
 	<artifactId>new-relic</artifactId>
-	<version>1.0.1</version>
+	<version>1.0.2-SNAPSHOT</version>
```

Comment:
@dalvizu as we don't push this plugin to any maven repos, and because integrating this PR would create an actual release, please remove the `SNAPSHOT` designation.

Reply:
Will remove - I'm not familiar with your release process so was trying to be defensive about not clobbering my change with something else. I usually use maven release plugin which encourages this.